.github/workflows/main.yml

```diff
@@ -103,12 +103,18 @@ jobs:
           REGISTRY_PASSWORD: ${{ secrets.DOCKERHUB_TOKEN }}
           EPINIO_TIMEOUT_MULTIPLIER: 3
         run: |
-          make acceptance-cluster-delete
```

Comment:
We put the cleanup before a job runs to be able to inspect the cluster in case of failures. Personally I never had to do that so I'm fine with this change. I don't know if anyone else found it useful though.

Reply:
Hmm and then we added a second cleanup below :D. I guess nodoby ever debugged a CI cluster then. All good.

Reply:
We added twice because there was some suspicions one time about k3d cluster still running and causing issues. Not sure that this is a good idea to keep a k3d cluster as the runners are shared for other tests now (rke, UI for example).